data/deep missions.txt

```diff
@@ -0,0 +1,1366 @@
+				goto next
+				
+			label friendly
+			`	The scientists look very surprised, and ask that you tell them what you found. You recall your encounter with the Remnant, a group of friendly humans who fled human space during the Alpha Wars and have advanced independently of humanity for centuries, and how they call the region of space the "Ember Waste."`
+			
+			label next
+			`	The group of scientists looks captivated the entire time, almost like kindergarten children listening to a fairy tale. When you finish talking, they immediately chatter among themselves, seeming to be most interested in the human beings which inhabit the region. They seem to agree that they need to learn more about these humans, and more specifically about the technology which they have developed.`
+			`	They eventually come to the conclusion that you should return to this region of space and scan the ships of these humans so that they can learn more about them.`
+			choice
+				`	"Okay, I'm willing to do it."`
+					accept
+				`	"This seems too risky for me. You'll need to find someone else."`
+					decline
+					
+	on accept
+		event "clear arculus"
+	
+	npc "scan outfits"
+		government Remnant
+		personality staying uninterested
+		fleet
+			names remnant
+			variant
+				"Albatross"
+				"Starling"
+		dialog `You have finished scanning the Remnant ships in the system. You may now return to <destination> with the scanner logs.`
+	
+	on fail
+		event "reset arculus"
+	
+	on complete
+		event "reset arculus"
+		event "deep: scan log research" 21
+		dialog
+			`The team is visibly excited when you return with the scanner logs. "Give us a few weeks to look these over," Ivan says, "then we will contact you about what we want you to do next."`
+
+
+
+event "clear arculus"
+	system Arculus
+		remove fleet "Large Remnant"
+		remove fleet "Small Remnant"
+		
+event "reset arculus"
+	system Arculus
+		add fleet "Large Remnant" 1500
+		add fleet "Small Remnant" 1000
+
+event "deep: scan log research"
+
+
+
+mission "Deep: Remnant Technology"
+	landing
+	name `Travel to <planet>`
+	description `Meet up with Ivan on <destination> to learn what he wants you to do next.`
+	source 
+		government Republic "Free Worlds" Syndicate Neutral Independent
+	destination Valhalla
+	to offer
+		has "event: deep: scan log research"
+	on offer
+		payment 250000
+		dialog `You recieve a message from Ivan after you land. "Thanks for all your help recently. We brought our findings to the Deep government and they gave us more funding! We've transfered 250,000 credits to your account and informed the Deep government of your recent help in this immense discovery. Please meet us on <destination> if you wish to assist us further. The help would be appreciated, but it is no longer required for us to continue our work."`
```

Comment:
\`You recieve a message from Ivan after you land. "Thanks for all your help recently. We brought our findings to the Deep government and they gave us more funding! We've transfered 250,000 credits to your account and made sure you were listed as a primary contributor on all the reports of these immense discoveries. Please meet us on \<destination\> if you wish to assist us further. Your help would be appreciated, but we will make progress even if you're a bit busy at the moment."\`